README.md

```diff
@@ -29,32 +31,23 @@ Increase `ulimit` to avoid such errors. For example,
 #### Sub-modules:
```

Comment:
In master branch we moved some jberet modules up one level (to be the same level as jsr352). We can rename the title of this section to be "JBeret Modules" and include all these modules whether they are under jsr352 or one level up.

jberet-support: https://github.com/jberet/jberet-support
jberet-rest-api: https://github.com/jberet/jberet-rest
jberet-ui: https://github.com/jberet/jberet-ui
...

We should also add a link to https://github.com/quarkiverse/quarkus-jberet to the above list.

We can remove links to jberet-distributioin, jberet-jpa-repository.

Reply:
Thanks for the inputs :) done the changes.